interop-testing/src/main/java/io/grpc/testing/integration/XdsTestServer.java

```diff
@@ -138,6 +147,10 @@ private void parseArgs(String[] args) {
         enableCsmObservability = Boolean.valueOf(value);
       } else if ("server_id".equals(key)) {
         serverId = value;
+      } else if ("address_type".equals(key)) {
+        if (!value.isEmpty()) {
```

Comment:
If the value is empty, we want to fail. What is the benefit of silently ignoring it?

Reply:
If they don't put anything it will fail.  if they explicitly put empty string as an argument then they get the default.

Reply:
But I don't really care, so will remove it.

Reply:
They didn't get the default. They got a noop. If they had `--address_type=IPV6 --address_type=` then they'd get IPV6.